src/components/PodcastCard/PodcastCard.js

```diff
@@ -4,43 +4,86 @@ import PropTypes from 'prop-types'
 PodcastCard.propTypes = {
   image: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
+  author: PropTypes.string.isRequired,
 }
 
-export default function PodcastCard({ image, title }) {
+export default function PodcastCard({
+  image,
+  title,
+  author,
+  id,
+  onClickDetails,
+}) {
   return (
     <Card>
-      <Image
-        src={image}
-        alt={`Podcast Picture of ${title}`}
-        width="150px"
-        height="150px"
-      />
-      <Title>{title}</Title>
+      <ImageContainer>
+        <Image src={image} alt={`Podcast `} width="150px" height="150px" />
+      </ImageContainer>
+      <InnerLayoutContainer>
+        <Title>{title}</Title>
+        <Author>{author}</Author>
+      </InnerLayoutContainer>
+      <SVG
+        onClick={() => onClickDetails(id)}
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+        width="100%"
+        height="100%"
+        viewBox="0 0 20 20"
+      >
+        <path d="M10 12c-1.105 0-2-0.895-2-2s0.895-2 2-2v0c1.105 0 2 0.895 2 2s-0.895 2-2 2v0zM10 6c-1.105 0-2-0.895-2-2s0.895-2 2-2v0c1.105 0 2 0.895 2 2s-0.895 2-2 2v0zM10 18c-1.105 0-2-0.895-2-2s0.895-2 2-2v0c1.105 0 2 0.895 2 2s-0.895 2-2 2v0z"></path>
+      </SVG>
     </Card>
   )
 }
 
 const Card = styled.section`
+  display: grid;
+  grid-template-columns: 3rem 1fr 1rem;
+  height: 3rem;
+  gap: 0.5rem;
+  width: 100%;
+  padding: 0 1.5rem 0 1rem;
+`
+const ImageContainer = styled.div`
+  height: 3rem;
+  width: 3rem;
+  align-self: flex-start;
```

Comment:
ImageContainer is a child of a grid right? So `start` would be the correct value for `align-self`
[https://css-tricks.com/snippets/css/complete-guide-grid/](https://css-tricks.com/snippets/css/complete-guide-grid/)